hypervisor/bsp/sbl/platform_acpi_info.c

```diff
@@ -10,13 +10,13 @@ const struct acpi_info host_acpi_info = {
 	6,				/* x86 family: 6 */
 	0x5C,				/* x86 model: 0x5C, ApolloLake */
 	{
-		{SPACE_SYSTEM_IO, 20, 0, 3, 0x400},	/* PM1a EVT */
+		{SPACE_SYSTEM_IO, 32, 0, 3, 0x400},	/* PM1a EVT */
 		{SPACE_SYSTEM_IO,  0, 0, 0,     0},	/* PM1b EVT */
-		{SPACE_SYSTEM_IO, 10, 0, 2, 0x404},	/* PM1a CNT */
+		{SPACE_SYSTEM_IO, 16, 0, 2, 0x404},	/* PM1a CNT */
 		{SPACE_SYSTEM_IO,  0, 0, 0,     0},	/* PM1b CNT */
 		{0x05,	0,	0},			/* _S3 Package */
 		{0x07,	0,	0},			/* _S5 Package */
-		(uint32_t *)0x7AEDCEFC,			/* Wake Vector 32 */
-		(uint64_t *)0x7AEDCF08			/* Wake Vector 64 */
+		(uint32_t *)0x7A86BC9C,			/* Wake Vector 32 */
+		(uint64_t *)0x7A86BCA8			/* Wake Vector 64 */
 	}
```

Comment:
define 2 MACROs seperately? and put that MACRO at platform specific place... For UEFI platfrom, do we have a method?

Reply:
All data in platform_acpi_info.c is hard-coded and the file is supposed to be generated by acpi offline tool. If here need a MACRO for wake vector only, then other data in the file might looks strange without MACRO... This is my understanding.
I've changed the struct initialization style to make code more readable in misra cleanup patch,
      i.e.      ".wake_vector_32 = (uint32_t *)0x7A86BC9CUL,"
How do you think? Is MACRO still needed?

Reply:
OK, but please send PR together with the MISRA-C cleanup patch. We don't want to bring new MISRA-C issues for new patch